#
# obstaclesName: list of strings,  name of the obstacle id ( corresponds to index order in obstaclesId)
#
# obstaclesId: list of integer, id with the identity of the class id.
#
# obstacleData: list of custom data structure for obstacles; 
#
#

string[] obstaclesName
uint32[] obstaclesId
ObstacleMapData[] obstaclesData